src/app/courses/courses.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="courses">
+    <div class="container">
+        <app-search></app-search>
```

Comment:
I believe, that it might be placed on app layout level...

Reply:
I suppose this component is used only on list of courses. I don't think this is necessary, for example, in course edit page or something else where we don't need this search. Therefore I put search component here.